Features3/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Delete a contact
+    As a user 
+    I want to delete contact 
+    So that contact will be removed from contact list
+
+    Background:
+        Given the User has logged in and navigated to Contact Fox page
+
+    Scenario Outline: Delete all created contacts from contact list
+        Given Contacts has been added with name as "<name>", email as "<email>", phone as "<phone>" and  contact type as "<contact-type>"
+        When User deletes a contact with email "<email>"
+        Then that contact with email "<emai>" should be removed from contact list 
+        And contact list should be decremented by 1
```

Comment:
i think this step is not required here since we are creating one user and deleting the same user.